index.html

```diff
@@ -0,0 +1,109 @@
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="main-content">
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">
+        Thank you for taking the time to help us improve the platform
+      </p>
+
+      <div class="name">
+        <label for="name" id="name-label">Name</label>
+        <input
+          type="text"
+          id="name"
+          name="name"
+          placeholder="Enter your name"
+          required
+        />
+      </div>
+
+      <div class="email">
+        <label for="email" id="email-label">Email</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="Enter your Email"
+          required
+        />
+      </div>
+
+      <div class="age">
+        <label for="number" id="number-label" class="form-control"
+          >Age(optional)</label
+        >
+        <input
+          type="number"
+          name="age"
+          id="number"
+          min="10"
+          max="99"
+          placeholder="age"
+        />
+      </div>
+
+      <div class="role">
+        <h3 class="titlee">Which option best describes your currnt role?</h3>
+        <select name="roles" id="dropdown">
+          <option value="Select an option">Select current role</option>
+          <option value="Student">Student</option>
+          <option value="Full Time Job">Full Time Job</option>
+          <option value="Full Time Learner">Full Time Learner</option>
+          <option value="Not To Say">Prefer Not To Say</option>
+          <option value="Other">Other</option>
+        </select>
+      </div>
+
+      <div class="radio-btn">
+        <h3 class="titlee">Would you recommend freeCodeCamp to a friend?</h3>
+        <div class="definitely">
+          <div>
+            <input type="radio" name="favorite" value="definitely" id="" />
+          </div>
+          <label for="definitely" checked>Definitely</label>
+        </div>
+        <div class="maybe">
+          <div><input type="radio" name="favorite" value="maybe" id="" /></div>
+          <label for="maybe">Maybe</label>
+        </div>
+      </div>
+
+      <div class="check-btn"></div>
+      <h3 class="titlee">
+        What would you like to see improved? (Check all that apply)
+      </h3>
+      <div class="frontendC">
+        <div>
+          <input
+            type="checkbox"
+            name="frontend1"
+            value="frontend"
+            id="frontends"
+          />
+        </div>
+        <label for="frontends">Front-end Projects</label>
+      </div>
+      <div class="backendC">
+        <div>
+          <input
+            type="checkbox"
+            name="backend1"
+            value="backend"
+            id="backends"
+          />
+        </div>
+        <label for="backends">Back-end Projects</label>
+      </div>
+      <textarea name="" id="" cols="30" rows="10"></textarea>
```

Comment:
Please kindly wrap the input elements in the HTML form element. For you to have a cleaner code

Reply:
- [ ] Kindly include an explanation, a "why", for each requested change. For example:`Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
- [ ] It is important that you are detailed enough when stating the reason you are requesting a change to completely assist the student in making those changes :+1: